test/modules/TestResultCoordinator/Reports/EdgeHubRestartTest/EdgeHubRestartMessageReport.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator.Reports.EdgeHubRestartTest
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Net;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    /// <summary>
+    /// This is a counting report to show test result counts, e.g. expect and match counts; and contains a list of unmatched test results.
+    /// </summary>
+    class EdgeHubRestartMessageReport : TestResultReportBase
+    {
+        public EdgeHubRestartMessageReport(
+            string trackingId,
+            string resultType,
+            bool isPassing,
+            long passedMessageCount,
+            Dictionary<string, ulong> messageCount,
+            Dictionary<HttpStatusCode, List<TimeSpan>> completedStatusHistogram,
```

Comment:
Remove all statistics in both report; it should be covered by metric.

Reply:
How does one introduce metric here? Metric does not know about the condition of the test. It only cares with at the scope of the module, not the test conditions.  Plus, this value will be available on LogAnalytics, right? you can just treat them like stat in the report.

Reply:
Would you please tell me what is not covered by metric?  And this will increase the size of report; which currently histogram is not our test focus now.

Reply:
Please remove "completedStatusHistogram" and below parameters; you can write these data to log in report generator.